### ISSUE TITLE
Choose custom interpreter at launch

### ISSUE THREAD

**Issue by kcharlie2** [spyder-ide/spyder#16299]
## Feature Description

I've seen a few requests for the ability to choose a custom interpreter at launch (instead of `Preferences -> Python interpreter -> Use the following Python interpreter` after launch), but I have not seen an Issue specifically for it.

This would help users separate their project (or other) environments from the spyder installation environment. 

User "frafra" suggested this feature at the end of this thread:
https://github.com/spyder-ide/spyder/issues/11362

I've also seen a user recently ask about this on SO, to which I replied with my hack for changing the config files:
https://stackoverflow.com/questions/68893798/spyder-set-python-interpreter-at-launch/68957981#68957981

**Comment by ccordoba12** [comment]
Hey @kcharlie2, thanks for the suggestion. This is an interesting idea, although it could be confusing to new users. I mean, demanding of them to understand what an interpreter is and why there's a need to select one.

I think the best would be to simply add the ability to associate an interpreter per project, which (in my mind) would be easier to explain to users. What do you think?

**Comment by bnavigator** [comment]
How about being able to select pre-configured environments/interpreters in the statusbar?

![image](https://user-images.githubusercontent.com/4623504/131848972-7c0225bb-b86b-44b2-ba94-e244ea7d2199.png)

**Comment by ccordoba12** [comment]
Yeah, that's something that we plan to add in the following months. Although not from the status bar, but from the Consoles menu.
